### PR TITLE
Document how to get an app running in our cluster

### PR DESCRIPTION
**Pull request by thockin**
As I wrote this, I realized I failed one step myself, so this also add
k8s.io RBAC groups to gke-security-groups@

### CODE REVIEW

**Comment by puerco** [comment]
Thanks @thockin this doc was really necessary

File: running-in-community-clusters.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+
+### DNS
+
+If you are exposing a public IP, you probably want DNS for it.  You can send a
+PR against [the DNS config](/dns/zone-configs/) to add or update your name.
+
+### TLS
+
+If you are exposing a public IP, you *must* do so securely.  For HTTP, this
+means TLS.  We run cert-manager in the cluster(s), so you can simply create a
```
Comment (BenTheElder) [inline running-in-community-clusters.md]: Future enhancement: link to cert-manager?

File: running-in-community-clusters.md [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+If you need to do that, talk to cluster-admins.
+
+## Configure monitoring and alerting
+
+Coming soon.  This is not yet fully decided.
+
+## Be responsible
+
+It is your responsibility to keep your workloads running and secure, and to use
+resources responsibly.  The cluster-admins team will provide some basic
```
Comment (BenTheElder) [inline running-in-community-clusters.md]: Do we mention anywhere how to reach cluster admins?

**Comment by thockin** [comment]
All Ben's comments addressed.

File: running-in-community-clusters.md (hunk 2 above)
Comment (BenTheElder) [reply running-in-community-clusters.md]: we do now. can't mark this as resolve though apparently.

**Comment by BenTheElder** [comment]
/lgtm
/hold cancel
thanks for writing this!

**Pull request merged**